buildconfig/pygame-stubs/mask.pyi

```diff
@@ -0,0 +1,40 @@
+from typing import Optional, Union, Tuple, List
+
+from pygame.math import Vector2
+from pygame.surface import Surface
+from pygame.rect import Rect
+from pygame.color import Color
+
+_Coordinate = Union[Tuple[float, float], List[float], Vector2]
+_ColorValue = Union[Color, Tuple[int, int, int], List[int], int, Tuple[int, int, int, int]]
+_RectValue = Union[Rect, Union[Tuple[int, int, int, int], List[int]], Union[Tuple[_Coordinate, _Coordinate], List[_Coordinate]]]
+
+
+def from_surface(surface: Surface, threshold: Optional[int]=127) -> Mask: ...
+def from_threshold(surface: Surface, color: _ColorValue, threshold: Optional[_ColorValue]=(0, 0, 0, 255), other_surface: Optional[Surface]=None, palette_colors: Optional[int]=1) -> Mask: ...
+class Mask:
+    def __init__(self, size: Union[List[int], Tuple[int, int]], fiil: Optional[bool]=False) -> None: ...
+    def copy(self) -> Mask: ...
+    def get_size(self) -> Tuple[int, int]: ...
+    def get_rect(self, **kwargs) -> Rect: ...  # Dict type needs to be completed
+    def get_at(self, pos: Union[List[int], Tuple[int, int]]) -> int: ...
+    def set_at(self, pos: Union[List[int], Tuple[int, int]], value: Optional[int]=1) -> None: ...
+    def overlap(self, othermask: Mask, offset: Union[List[int], Tuple[int, int]]) -> Union[Tuple[int, int], None]: ...
+    def overlap_area(self, othermask: Mask, offset: Union[List[int], Tuple[int, int]]) -> int: ...
```

Comment:
I guess offset can be any sequence of 2 ints, not just lists, and tuple?

Reply:
I can add it, but then it would accept tuple with any number of ints (because you can't check length of regular sequence)